chain-abci/src/app/app_init.rs

```diff
@@ -226,9 +238,7 @@ fn check_and_store_consensus_params(
             // but it should be migrated to "time", in which case this check will make sense
             // (as unbonding time is in seconds, not blocks)
             warn!("consensus parameters not checked (TODO)");
-            inittx.put(
-                COL_EXTRA,
-                b"init_chain_consensus_params",
+            storage.store_consensus_params(
```

Comment:
New version breaks storage transaction into smaller ones, not sure is that correct?

Reply:
it just hides this detail -- later some of it may be changed when this is fixed: https://github.com/crypto-com/chain/issues/885